### ISSUE TITLE
Rack segfaults when attempting to load a plugin generated via faust

### ISSUE THREAD

**Issue by jazamatronic** [ohmtech-rdi/eurorack-blocks#460]
G'day,

Rack is instantly crashing when trying to load a plugin that was built by erbb from faust sources - such as the flanger example.

I have fixed it locally [here](https://github.com/ohmtech-rdi/eurorack-blocks/compare/main...jazamatronic:eurorack-blocks:rack_segfault?expand=1#diff-3fea5fd94ce514fc3d31e7eb873a06e9d46e053cee88f8cefcd4534b8810d2c8) by initializing dsp memory during the init.

I can create a pull request if you'd like?

Cheers

**Comment by ohmtech-rdi** [comment]
Hi @jazamatronic !

Thanks a lot for your patch. I'm not sure how it could possibly work before!
I'll create the Pull request.

Thanks!

**Comment by ohmtech-rdi** [comment]
Ah so apparently, it depends on the Faust version. The Faust version doesn't generate the `memoryCreate` function for at least version 2.20.2, which is the current version `apt`:

```
Get:16 http://azure.archive.ubuntu.com/ubuntu focal/universe amd64 libfaust2 amd64 2.20.2~ds0-1 [1838 kB]
Get:17 http://azure.archive.ubuntu.com/ubuntu focal/universe amd64 faust-common all 2.20.2~ds0-1 [29.4 MB]
Get:18 http://azure.archive.ubuntu.com/ubuntu focal/universe amd64 faust amd64 2.20.2~ds0-1 [4071 kB]
```

The version I have locally is:

```
$ faust --version
FAUST Version 2.41.1
```

And this one actually generates the `memoryCreate` function.

Not sure what to do, a bit surprised a breaking change was introduced, but I don't know all the mechanics behind.

@sletz Could you please help us with that? Thanks!

**Comment by jazamatronic** [comment]
G'day @ohmtech-rdi,

Thanks for taking a look.  I'm running a very new clone/build of faust, running on Linux:

```
$ faust --version
FAUST Version 2.50.6
```

Cheers

**Comment by ohmtech-rdi** [comment]
Thanks for letting me know your version, indeed that's very new, but I can already reproduce the problem with a stock `faust` build using `brew` on macOS 10.15 so that's going to be problematic as well for every new users.

I guess there might be a way to keep backward compatibility, will check with @sletz

**Comment by ohmtech-rdi** [comment]
- Closed in favor of #464 (It's not possible to rewrite history for gh PRs, and I needed to correct style on first commit)
